lapce-data/src/data.rs

```diff
@@ -335,15 +335,21 @@ impl LapceData {
         Ok(())
     }
 
-    pub fn check_local_socket(paths: Vec<PathBuf>) -> Result<()> {
+    pub fn check_local_socket(paths: &Vec<PathBuf>) -> Result<()> {
```

Comment:
```rs
warning: writing `&Vec` instead of `&[_]` involves a new object where a slice will do
   --> lapce-data\src\data.rs:338:38
    |
338 |     pub fn check_local_socket(paths: &Vec<PathBuf>) -> Result<()> {
    |                                      ^^^^^^^^^^^^^ help: change this to: `&[PathBuf]`
    |
    = note: `#[warn(clippy::ptr_arg)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#ptr_arg
```